caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -687,22 +723,46 @@ func parseLog(h Helper) ([]ConfigValue, error) {
 					return nil, h.ArgErr()
 				}
 
+			case "include":
+				if !h.NextArg() {
+					return nil, h.ArgErr()
+				}
+				for h.NextArg() {
+					cl.Include = append(cl.Include, h.Val())
+				}
+
+			case "exclude":
+				if !h.NextArg() {
+					return nil, h.ArgErr()
+				}
+				for h.NextArg() {
+					cl.Exclude = append(cl.Exclude, h.Val())
+				}
```

Comment:
I don't think we want to add `include` and `exclude` to the `log` directive, but we would want it for the global option. So that would probably need to be wrapped with that boolean.

Reply:
Makes sense, that was one of the things I was wondering about, I can disable these for the global case.

Reply:
To be clear - we want to _allow_ `include` and `exclude` for globals, but _disallow_ it for the `log` directive. The `log` directive's purpose is to configure access logs, so the `include` part of it is already "managed".